site/src/styles/markdown.scss

```diff
@@ -79,8 +80,7 @@ $link-icon-margin: 12px;
   }
 
   :global(.md-h2) {
-    font-size: 2rem;
-    line-height: 1.125;
+    @include kz-typography-heading-2;
```

Comment:
I didn't use the Text component here because we need to be able to support Markdown which doesn't let you add css classes, so you need to do complicated CSS selectors, and you can't reach into CSS Modules obfuscated class names (which are used in our Text component). Eg:
https://github.com/cultureamp/kaizen-design-system/blob/master/site/src/styles/markdown.scss#L88

Reply:
It's a bit of an issue that we can't use our Text components in Kaizen Site, but I.M.O Kaizen Site is not what the Design System was designed for, and Markdown comes with a lot of complications. It's relatively small and maintainable if we have to do some custom stuff to over come limitations of the Design System.